### PR TITLE
Add icon for coming `notes` feature

### PR DESCRIPTION
**Pull request by wiktor-obrebski**
Add icon for coming `notes` feature.
Needed for https://github.com/DFHack/scripts/pull/1287

### CODE REVIEW

File: data/art/note-green-pin.png
Comment (myk002) [inline data/art/note-green-pin.png:1]: could you rename this file `green_pin_map.png`? we're trying to give assets intended for the map layer a `_map.png` suffix.

Comment (wiktor-obrebski) [reply data/art/note-green-pin.png:1]: I can, but I have doubts about generality of the name.
`green_pin_map.png` suggest that other tool can use it on the map - but in reality other tools would probably need its own version with the pin placed in different relative position to the tile.

Comment (myk002) [reply data/art/note-green-pin.png:1]: as long as it has the `_map` suffix, it's fine. `note_green_pin_map.png` would be ok too

Comment (wiktor-obrebski) [reply data/art/note-green-pin.png:1]: done

**Review by myk002 — APPROVED** [review]

**Pull request merged**